src/Microsoft.VisualStudio.Editors/AddImportsDialogs/AddImportDialogBase.vb

```diff
@@ -4,8 +4,11 @@ Imports System.Drawing
 Imports System.Windows.Forms
 Imports VsShell = Microsoft.VisualStudio.Shell.Interop
 Imports VsErrorHandler = Microsoft.VisualStudio.ErrorHandler
+Imports System.ComponentModel
 
 Namespace Microsoft.VisualStudio.Editors.AddImports
+
+    <TypeDescriptionProvider(GetType(AbstractControlTypeDescriptionProvider(Of AddImportDialogBase, Form)))>
```

Comment:
Yeah, the designer needs to create an instance of the base class to be able to design it, in this case - the base is abstract, so at design type we swap it with "Form"